src/bittybuzz/include/bittybuzz/BittyBuzzSystem.h

```diff
@@ -41,6 +42,11 @@ namespace BittyBuzzSystem {
      **/
     extern IBittyBuzzMessageService* g_messageService;
 
+    /**
+     *@brief BSP used by the bbvm for random numbers
+     **/
+    extern IBSP* g_bsp;
```

Comment:
Why the global variable? Could you change it to a static reference to the value obtained from the BSP:Container?

Reply:
See the other comment.